hw/ip/flash_ctrl/dv/cov/flash_ctrl_cov.vRefine

```diff
@@ -0,0 +1,59 @@
+    </ccf-files>
+  </information>
+  <rules>
+    <rule ccType="type" domain="icc" entityName="prim_generic_flash/tck_i" entityType="toggle" excTime="1652803318" name="exclude" packageName="worklib" reviewer="unknown" user="0" vscope="default"></rule>
```

Comment:
the jtag ports (tck_i, tdi etc) really should have a test case that just wiggles them at the level of flash_ctrl and an observe inside the prim_flash to make sure the connectivity is right. See my original comment, this should not be waived.

Reply:
JTAG is different domain and we did not take at into consideration to write a test.

Reply:
Ok, I will remove that from exclusion.